eng/scripts/docs/Docs-ToC.ps1

```diff
@@ -32,7 +61,11 @@ function Get-python-DocsMsTocData($packageMetadata, $docRepoLocation) {
     $readmeMetadata = &$GetDocsMsMetadataForPackageFn -PackageInfo $packageMetadata.FileMetadata
     $packageLevelReadmeName = $readmeMetadata.DocsMsReadMeName
   }
+  return $packageLevelReadmeName
+}
 
+function Get-python-DocsMsTocData($packageMetadata, $docRepoLocation) {
+  $packageLevelReadmeName = Get-python-PackageLevelReadme -packageMetadata $packageMetadata
```

Comment:
Similar comment as this https://github.com/Azure/azure-sdk-for-java/pull/29818#discussion_r917525691

If this is the only place it's used, we may not need to define the generic function name in common.ps1